CHANGELOG.txt

```diff
@@ -6,9 +6,10 @@ Pyomo CHANGELOG
 Current Development
 -------------------------------------------------------------------------------
 
-- Resolved Python 3.4 build errors on Appveyor.
+- Resolved Python 3.4 build errors on Appveyor. (#438)
```

Comment:
This change s a bleed over from #438.  At a minimum, we need to resync with master.

Reply:
I already synced with master.  This documentation was omitted in that PR.  I figured it was a waste of time to have a PR for 7 characters.